compiler/one-cmds/onelib/utils.py

```diff
@@ -187,6 +187,48 @@ def safemain(main, mainpath):
         sys.exit(255)
 
 
+def run_ret(cmd, *, one_cmd: str = None, err_prefix=None, logfile=None):
```

Comment:
i want to know what it is used for? usage scenario?

Reply:
you can read scenario at #13886.
existing `run()` will exit the program if fails.
`one-import-onnx` wants to check the return code for next process.